app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -63,6 +63,26 @@ class AdvancedFilter extends React.Component {
         advancedFilterOptions[Number(filterIndex)].options = formattedOptions;
       });
 
+    // Dynamically populate common exposure cohort name/description and location options
```

Comment:
I think we should sort the population of these values alphabetically. Currently it feels a little random:
![image](https://user-images.githubusercontent.com/52747882/142060726-7fcf5d2e-003f-45ef-9792-8bc4e08cf4cb.png)

Reply:
This has changed into a multi select field where users can type to find options (which I think solves the ui problem). The options are sorted in the order of how common/frequent these values are present within each jurisdiction.

Reply:
I agree that these should still be alphabetical.  You have a search feature with the multi select so scrolling for something shouldn't be an issue

Reply:
This is not something that I decided on, please discuss with the PO team if you disagree